test/metabase/api/downloads_exports_test.clj

```diff
@@ -958,37 +957,20 @@
                     :subscription-attachment (first subscription-result)}))))))))
 
 (deftest downloads-row-limit-test
-  (testing "Downloads row limit works."
-    (mt/with-temporary-setting-values [public-settings/download-row-limit 105]
+  (testing "Downloads row limit respects minimum (#52019)"
```

Comment:
If I understand this test correctly it's just asserting that more than 100 rows are returned even when the download row limit is set below the minimum?

Can we still have a test that ensures that it can be _raised_? If you make the minimum row count a dynamic var (see comment above) then you can bind that to a lower value in the tests